### ISSUE TITLE
"Drag window to screen edge to snap" causes stuttering while dragging-n-dropping in Godot Engine

### ISSUE THREAD

**Issue by craigostrin** [rxhanson/Multitouch-Community#166]
So, this is probably an incredibly niche issue, but it seems worth reporting nonetheless.

Basically, with "Drag to screen edge" enabled, trying to drag literally anything in the Godot Engine editor will cause massive stuttering. This includes dragging the edges of window panes to resize them, dragging files, dragging a text selection box... Literally everything.

Interestingly, this only applies when the left-mouse-button is held down. If you drag an item without holding down left-mouse, everything is smooth. (You can make this happen either because the engine skipped a frame and didn't realize you released the button to drop the item you're dragging, or you can do it by coding a game object that uses drag-n-drop without needing to hold left-mouse.)

[Here's a link](https://github.com/godotengine/godot/issues/39608#issuecomment-1647638152) to the bug report I left on Godot's side, which is much more detailed and includes videos.

This isn't a big deal for me, since I use Rectangle for window management anyway. So disabling "Drag window to screen edge to snap" doesn't impact my life at all. But maybe this will be a simple fix or at least will help warn other Godot+Multitouch users who might run into the same issue.

**System info**

I'm on a Macbook Pro M2 Max with MacOS 13.4.1

Let me know if you'd like any more info from my end.

**Comment by rxhanson** [comment]
Thanks for reporting this. It sounds like you don't experience this while using Rectangle? Certain apps have this issue, even with Rectangle, but if you only experience this in Multitouch then I'll dig in and see what I can fix.

**Comment by craigostrin** [comment]
Yeah, that's correct. I have the same functionality enabled in Rectangle and it doesn't affect Godot. Thanks for taking a look!
